recipes/pytng/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "pytng" %}
+{% set version = "0.2.2a1" %}
+{% set sha256 = "688fb4afcc5630bd98553834b5dbd69eb04b83e2fc23f4322986f52add5bcfda" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }} 
+
+build:
+  skip: true  # [win and py2k]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    
+  host:
+    - python
+    - pip
+    - cython
+    - numpy
```

Comment:
If you're building against `numpy` see:

https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy